File: 3516-find-closest-person/3516-find-closest-person.java
class Solution {
    public int findClosest(int x, int y, int z) {
        int d=Math.abs(z-x);
        int di=Math.abs(z-y);
        if(d==di)return 0;
        if(d<di)return 1;
        return 2;
    }
}

File: 3516-find-closest-person/3516-find-closest-person.py
class Solution:
    def findClosest(self, x: int, y: int, z: int) -> int:
        w1=abs(x-z)
        w2=abs(y-z)
        if w1<w2:
            return 1
        elif w1>w2:
            return 2
        else:
            return 0